packages/botonic-core/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@botonic/core",
-  "version": "0.19.0",
+  "version": "1.0.0-dev.0",
```

Comment:
Should all packages be in version `1.0.0-dev.1`?

Reply:
Atm I and as we are going to release more often dev releases, I prefer to leave it like this, because in this way packages that are dependant on this one don't need to have anything changed as they are already pointing to 'dev' version.

Reply:
Seems good then!